### PR TITLE
Command to fetch/update issue, and fetch event

### PR DESCRIPTION
**Pull request by gugsrs**
Added command to fetch issue, to update issue status and to fetch an event.
I've updated the README.rst with theses new commands and added a table of contents.
And updated the tests to use CliRunner.

I'll open another PR with a command to list the organization projects, and improve command outputs to be more descriptive, maybe print the json when fetching only one issue. :thinking:

### CODE REVIEW

File: patrol/cli.py [hunk 1]
```diff
@@ -20,10 +20,39 @@ def events(organization, project_name):
...
+@cli.command(help='Retrive issue')
+@click.argument('issue_id')
+def issue(issue_id):
+    issue = patrol.issue(issue_id)
+    click.echo('{id}: {title} - {status}'.format(**issue))
+
+
+@cli.command(help='Update issue')
+@click.argument('issue_id')
+@click.argument('status')
```
Comment (daneoshiga) [inline patrol/cli.py]: I believe here would be better to use a Choice Option http://click.pocoo.org/5/options/#choice-options

File: patrol/cli.py [hunk 2]
```diff
@@ -20,10 +20,32 @@ def events(organization, project_name):
...
 @cli.command(help='Lists project issues')
 @click.argument('organization')
 @click.argument('project_name')
 def issues(organization, project_name):
     click.secho('issues for {}'.format(project_name))
     for issue in patrol.issues(organization, project_name):
         click.echo('{id}: {title}'.format(**issue))
+
+
+@cli.command(help='Retrive issue')
```
Comment (daneoshiga) [inline patrol/cli.py:42]: I believe it's better to keep it as two commands.. less error-prone usage I believe, what do you think?

**Comment by daneoshiga** [comment]
The complete json for one issue is a bit too much information, I'm thinking in a way to filter it out, but probably when it's just one issue, makes sense to just print all of it by now.. fell free to do so

File: patrol/cli.py (hunk 2 above)
Comment (gugsrs) [reply patrol/cli.py:42]: Hm Okay! I agree with that, I just thought it would look better like this. I'll put it back as two commands.

**Comment by gugsrs** [comment]
I'll let the json part for another pr. :)

**Comment by daneoshiga** [comment]
Awesome, add an entry as 0.1.0 version in CHANGES.rst and I'll merge :)

**Comment by daneoshiga** [comment]
Thanks for your PR :)

**Pull request merged**